test/unit/dom/SVGElement.js

```diff
@@ -0,0 +1,46 @@
+import canPatchNativeAccessors from '../../../src/util/can-patch-native-accessors';
+import create from '../../lib/create';
+
+describe('dom: SVGElement', function () {
+  runTests('div');
+  runTests('slot');
+  runTests('host');
+  runTests('root');
+
+  function runTests(type) {
+    describe(`${type}: `, () => {
+      let host, root, slot, div, elem;
```

Comment:
it seems like the `host, root, slow, div` elements are created each time `runTests()` is run, but then only 1 of those vars is used each time? would be nice to only create those elements when they are needed.